libs/media-player/src/lib/audio-annotator.tsx

```diff
@@ -59,16 +59,21 @@ interface AudioAnnotatorProps {
     mimeType?: AudioMIMEType;
     selectedTimeRange: Nullable<TimeRangeSelection>;
     onTimeRangeSelected: (timeRangeSelected: Nullable<TimeRangeSelection>) => void;
+    /**
+     * This gives us an escape hatch from the React rerendering cycle and more
+     * freedom to syncronize state between the audio element and ad-hoc command
+     * forms.
+     */
+    audioRef: RefObject<HTMLAudioElement>;
```

Comment:
from my reading and prototyping the custom timeline, `useRef` can also be used for grabbing information about the layout of a div.  I'll be interested to see if that will work in our context.